ui/StatusQ/src/StatusQ/Core/Utils/ClippingWrapper.qml

```diff
@@ -0,0 +1,41 @@
+import QtQuick 2.15
+
+/*!
+   \qmltype ClippingWrapper
+   \inherits Item
+   \inqmlmodule StatusQ.Core.Utils
+   \since StatusQ.Core.Utils 0.1
+   \brief A component allowing to clip a nested component with a specified margins.
+*/
+Item {
+    id: root
+
+    default property alias data: content.data
+
+    property real clipTopMargin: 0
```

Comment:
do we need some documentation for these properties?

Reply:
Yeah, it was a bit unclear. I extended the description. I think it's clearer now.